trillian/ctfe/config.go

```diff
@@ -210,6 +214,28 @@ func ValidateLogConfig(cfg *configpb.LogConfig) (*ValidatedLogConfig, error) {
 		}
 	}
 
+	switch cfg.ExtraDataIssuanceChainStorageBackend {
+	case configpb.LogConfig_ISSUANCE_CHAIN_STORAGE_BACKEND_CTFE:
+		// Validate the combination of CtfeStorageConnectionString and ExtraDataIssuanceChainStorageBackend
+		if len(cfg.CtfeStorageConnectionString) == 0 {
+			return nil, errors.New("missing ctfe_storage_connection_string when issuance chain storage backend is CTFE")
+		}
+		// Validate CTFEStorageConnectionString
+		if len(cfg.CtfeStorageConnectionString) > 0 {
```

Comment:
You can ding this `if` - we know the len must be > 0 here because of the check on L220

Reply:
Good catch. Removed.